src/main/java/bio/terra/cli/command/shared/options/NewCloningInstructions.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import picocli.CommandLine;
+
+public class NewCloningInstructions {
+  /**
+   * Command helper class that defines the project id and dataset id when updating a BigQuery
+   * datasets or BigQuery data table.
+   *
+   * <p>This class is meant to be used as a @CommandLine.Mixin.
+   */
+
+  // Cloning option must have a different default for referenced resources (REFERENCE) than
```

Comment:
Could we have a comment about the update stuff?

```
// Can't have a default. This is used for update. Need `null` in case user does not want to update cloning instructions.
```

Reply:
down